contents/blog/brand.md

```diff
@@ -0,0 +1,128 @@
+---
+date: 2023-06-25
+title: How to brand your startup
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - james-hawkins
+category: CEO diaries
+---
+
+The world would be a lot more fun if most startups didn't seem to have experienced a personality vasectomy.
+
+I believe humans have evolved to conform - when we all lived in the wilderness, staying alive required you to fit in with a tribe, after all. Fast forward that background to today and, amongst other things (!), the majority of the world's software companies feel the same as each other.
+
+## There are good reasons for this, you idiot
+
+I bet there are. You want to get the Minimum Viable Product built and validated.
+
+You need to make money, you're going to fail without it.
+
+Creating a pretty website won't help with that.
+
+## But... will it?!
```

Comment:
per point above re: crossheads, i think we're also maybe using a couple too many right at the start. you could have a version of this back-and-forth conversation between the crossheads and the main text without crossheads. so...

Creating a pretty website won't help with that, right? Wrong.

When we knew that PostHog was something we wanted to launch, we questioned...

etc.